src/lib/context.go

```diff
@@ -40,6 +41,11 @@ type ArtifactInfo struct {
 	BlobMountDigest      string
 }
 
+// ProxyCacheRemoteRepo get the remote repository name, used in proxy cache
+func (a *ArtifactInfo) ProxyCacheRemoteRepo() string {
```

Comment:
All functions defined in this file are context related, but `ProxyCacheRemoteRepo` isn't

Reply:
+1 let's add a package `/src/pkg/proxycache

Reply:
Move to controller proxy  package